.github/workflows/main.yml

```diff
@@ -208,8 +209,9 @@ jobs:
       run: |
         C:/vcpkg/vcpkg install glslang[tools] --triplet=x64-windows-release
         C:/vcpkg/installed/x64-windows-release/tools/glslang/glslangValidator.exe -v
-        python python/Scripts/generateshader.py resources/Materials/Examples --target glsl --validator C:/vcpkg/installed/x64-windows-release/tools/glslang/glslangValidator.exe
-        python python/Scripts/generateshader.py resources/Materials/Examples --target essl --validator C:/vcpkg/installed/x64-windows-release/tools/glslang/glslangValidator.exe
+        python python/Scripts/generateshader.py ../resources/Materials/Examples --target glsl --validator C:/vcpkg/installed/x64-windows-release/tools/glslang/glslangValidator.exe
```

Comment:
Perhaps we don't need the `../` prefix to be added to these filepaths?  That might be a source of the new testing errors that we're seeing.